python/nav/models/manage.py

```diff
@@ -574,6 +575,15 @@ def get_environment_sensors(self):
             | Q(unit_of_measurement__icontains='percent')
         )
 
+    @property
+    def mac_addresses(self) -> Set[str]:
+        """Returns a set of collected chassis MAC addresses for this Netbox"""
+        macinfo_match = (Q(key="bridge_info") & Q(variable="base_address")) | (
```

Comment:
Whenever I see queries like "macinfo_match" I flashback to Relational Algebra and absurdly redundant tables after a join (to illustrate a point). .. and it becomes a thrill to see if and how the redundancy is removed..